src/Signup.js

```diff
@@ -63,10 +61,8 @@ function Signup() {
           });
   
           // firebase user collection (create a document)
-          await firebase
-            .firestore()
-            .collection('users')
-            .add({
+          await addDoc(collection(db, 'users'),
```

Comment:
Is it necessary to use await for addDoc since you already have await for the firebase?